arithmetization/src/main/java/net/consensys/linea/continoustracing/SlackNotificationService.java

```diff
@@ -65,12 +65,11 @@ public void sendCorsetFailureNotification(
                         section ->
                             section.text(
                                 markdownText(
-                                    "Block hash: *"
-                                        + blockHash
-                                        + "*\n\n"
-                                        + "Trace verification failed with the following error:\n\n"
+                                    "Trace verification failed with the following error:\n\n"
                                         + "```"
-                                        + validationResult.corsetOutput()
+                                        + validationResult
+                                            .corsetOutput()
+                                            .replaceAll("\u001B\\[[;\\d]*m", "")
```

Comment:
What does this do exactly? It might make sense to add a comment explaining this one line, because at least for me it is not clear without spending some time looking into it.

Reply:
It removes all the color/bold ANSI escape codes from Corset, because Slack refuses them